sdk/servicebus/Microsoft.Azure.ServiceBus/README.md

```diff
@@ -1,5 +1,7 @@
 # Azure Service Bus client library for .NET
 
+> Please note, a newer package [Azure.Messaging.ServiceBus](https://www.nuget.org/packages/Azure.Messaging.ServiceBus) for [Azure Service Bus](https://azure.microsoft.com/services/service-bus/) is available as of December 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the [migration guide](https://aka.ms/azsdk/net/migrate/sb) for more details.
```

Comment:
```suggestion
> Please be aware that a newer package, [Azure.Messaging.ServiceBus](https://www.nuget.org/packages/Azure.Messaging.ServiceBus) is available as of November 2020. While this package will continue to receive critical bug fixes, we strongly encourage upgrading for new development.  Please see the [migration guide](https://aka.ms/azsdk/net/migrate/sb) for more details.
```

Reply:
Was it November or December?   The README says one thing, the package description another.  😄